spec/requests/postscontroller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Postscontroller', type: :request do
+  describe 'GET #post' do
+    before(:each) do
+      get '/posts'
+    end
+
+    it 'GET request return correct responses' do
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
+      expect(response.body).to include('post index view')
+    end
+  end
+
+  describe 'Get #show' do
+    before(:each) do
+      get '/posts/798'
+    end
+
+    it 'return show post' do
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:show)
+      expect(response.body).to include('show post')
+    end
+  end
```

Comment:
- [ ] You tested for the index and show actions for the Post controller as required :+1:  unfortunately, all these tests are failing. Kindly make sure that your tests are passing. In case of any doubt, check out the screenshot below:


![Screenshot from 2022-04-29 01-51-34](https://user-images.githubusercontent.com/63087491/165869959-5a98b69d-ede2-4cca-bcb8-19bebcfb872c.png)

Reply:
resolved

Reply:
![Screenshot from 2022-04-30 10-22-15](https://user-images.githubusercontent.com/52419124/166100087-965073aa-0b5b-4246-8089-d9e62110ff84.png)